nimble/host/src/ble_hs_startup.c

```diff
@@ -40,7 +40,7 @@ ble_hs_startup_read_sup_f_tx(void)
     /* for now we don't use it outside of init sequence so check this here
      * LE Supported (Controller) byte 4, bit 6
      */
-    if (!(rsp.features & 0x0000006000000000)) {
+    if (!(rsp.features &  htole64(0x0000006000000000))) {
```

Comment:
Hi,

I think it would be cleaner to do other way around:
if (!(le64toh(rsp.features) & 0x0000006000000000))) {


that way constant is used in host order, while LE value is converted to host before use

Reply:
complete .